docs/data/tree-view/rich-tree-view/customization/IconExpansionTreeView.tsx

```diff
@@ -36,40 +34,14 @@ const MUI_X_PRODUCTS: TreeViewBaseItem[] = [
   },
 ];
 
-const CustomTreeItem = React.forwardRef(function MyTreeItem(
-  props: TreeItem2Props,
-  ref: React.Ref<HTMLLIElement>,
-) {
-  const { interactions } = useTreeItem2Utils({
```

Comment:
This hook could probably use a better documentation after removing this part 🤔

Reply:
Clearly, I added it to #13161